pubtools/_content_gateway/push_cgw.py

```diff
@@ -1,2 +1,196 @@
+import argparse
+from .cgw_client import *
+from .cgw_authentication import CGWBasicAuth
+from .utils import yaml_parser, validate_schema
+import logging
+
+LOG = logging.getLogger("pubtools.cgw")
+LOG_FORMAT = "%(asctime)s [%(levelname)-8s] %(message)s"
+logging.basicConfig(level=logging.DEBUG, format=LOG_FORMAT)
+
+
+class PushCGW:
```

Comment:
Are there no tests? We have a requirement to cover every newly line with unit tests.

Reply:
Yup! I'll add the tests as well but same like docstrings I want this to be confirmed, later I'll add the tests, docstrings and some extra layers of validation.